man/elixirc.1

```diff
@@ -0,0 +1,45 @@
+.Dd March 17, 2015
+.Dt ELIXIRC 1
+.Os
+.Sh NAME
+.Nm elixirc
+.Nd The Elixir compiler
+.Sh SYNOPSIS
+.Nm
+.Op Ar OPTIONS
+.Ar
+.Sh DESCRIPTION
+The compiler is intended for compilation one or more files containing the Elixir source code. The files should have the extension
+.Em .ex .
+.Sh OPTIONS
+.Bl -tag -width Ds
+.It Fl o Ar directory
+Places the output file in
+.Ar directory .
+If the directory is not specified, the current working directory will be used for the purpose.
+.It Fl -ignore-module-conflict
+Disables warnings when a module was previously defined.
+.It Fl -no-debug-info
+Turns off producing debugging information.
+.It Fl -no-docs
+Turns off generating documentation.
+.It Fl -warnings-as-errors
+Makes all warnings into errors.
+.It Fl -verbose
+Activates verbose mode.
+.It Fl -
+Separates the options passed to the compiler from the options passed to the executed code.
+.El
+.Sh ENVIRONMENT
+.Bl -tag -width Ds
+.It Ev ELIXIR_ERL_OPTIONS
```

Comment:
This is a general environment variable that could be documented on all command lines.